### ISSUE TITLE
This option are not responsive

### ISSUE THREAD

**Issue by rodolphebertozzo** [soflyy/breakdance-bugs#336]
Make this option responsive please, see the screenshoot if you would understand why it's very important
![suggest](https://user-images.githubusercontent.com/56270626/188499645-317b6df9-406a-4643-aa46-1858ad912672.png)

**Comment by sanjaypanikkar** [comment]
I agree. All gap controls and similar spacing controls need to be responsive.

**Comment by MrBushid0** [comment]
+1

**Comment by rodolphebertozzo** [comment]
@a-dubiel

**Comment by rodolphebertozzo** [comment]
Solved in Breakdance 1.1. So, I close this issue.
